Updated syntax for Get-AzureRmVm and Stop-AzureRmVm cmdlets

paulkel opened the pull request:
Updated syntax to change -VMName parameter to -Name for the Get-AzureRmVM and Stop-AzureRMVm cmdlets.

azurecla commented:
Hi __@benwllms918__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@benwllms918 We're closing this PR because it appears to have only been a test.  Please comment if we should reopen.

paulkel committed:
Updated syntax for Get-AzureRmVm and Stop-AzureRmVm cmdlets

Updated syntax to change -VMName parameter to -Name for the Get-AzureRmVM and Stop-AzureRMVm cmdlets.
